add solution to ex:192 (Week 4 homework) (#52)
benediktahrens/CT4P: tex/solutions.tex

	and in the same way, we have $ \projl \circ g \circ f = \projl $. Therefore, $ g \circ f = h = \Id[A \times B] $. In the same way, we have $ f \circ g = \Id[B \times A] $. Therefore, we conclude that $ f $ is the isomorphism we are looking for, with inverse $ g $.
\end{solution}

\begin{solution}[\cref{exer:natlist_is_initial}]\label{sol:natlist_is_initial}
		We construct a category $\Cat{L}$ as follows
		\begin{itemize}

> 			\item objects are triplets, made from a set $X$, an element $x \in X$ and a function $l : \NN \times X \to X$

Reviewer: ```suggestion
			\item objects are triples consisting of a set $X$, an element $x \in X$ and a function $l : \NN \times X \to X$
```

          1 & X && {\NN \times X} & X \\
          & {X'} && {\NN \times X'} & {X'}
          \arrow["x", from=1-1, to=1-2]
          \arrow["{x'}"', from=1-1, to=2-2]
          \arrow["f"{description}, from=1-2, to=2-2]
          \arrow["l"{description}, from=1-4, to=1-5]
          \arrow["{l'}"{description}, from=2-4, to=2-5]
          \arrow["f"{description}, from=1-5, to=2-5]
          \arrow["{1_\NN \times f}"{description}, from=1-4, to=2-4]
        \end{tikzcd}
      \]
			\item the identity morphisms $1_{(X, x, l)}$ is the identity function $1_X : X \to X$
			\item morphism composition is just function composition
		\end{itemize}
		\begin{thm}
			The identity function $1_X : X \to X$ makes the following diagrams commute
			\[
				\begin{tikzcd}
					1 & X && {\NN \times X} & X \\
					& X && {\NN \times X} & X
					\arrow["x", from=1-1, to=1-2]
					\arrow["{x}"', from=1-1, to=2-2]
					\arrow["{1_X}"{description}, from=1-2, to=2-2]
					\arrow["l"{description}, from=1-4, to=1-5]
					\arrow["{l}"{description}, from=2-4, to=2-5]
					\arrow["{1_X}"{description}, from=1-5, to=2-5]
					\arrow["{1_\NN \times 1_X}"{description}, from=1-4, to=2-4]
				\end{tikzcd}
			\]

> 			That is the following equations hold:

Reviewer: ```suggestion
			That is, the following equations hold:
```
